public/locales/fr/translation.json

```diff
@@ -187,7 +187,8 @@
       "hideTable": "Cacher le tableau attributaire",
       "noFeature": "Aucun objet à afficher pour cette couche.",
       "readMessage": "<0>Voir</0> cet objet.",
-      "readOrUpdateMessage": "<0>Voir</0> ou <2>Modifier</2> cet objet."
+      "readOrUpdateMessage": "<0>Voir</0> ou <2>Modifier</2> cet objet.",
+      "count": "{{ count }} résultats"
```

Comment:
I would name it `results :  "{{ count }} résultat"` and add a `result_plural :  "{{ count }} résultats"`